src/pages/api/index.tsx

```diff
@@ -0,0 +1,266 @@
+                />
+                <SectionWrapper
+                    className={styles.LargerContent}
+                    heading="API reference"
+                    description={<div className="MainSectionContent">
+                    The Apify API allows developers to interact programmatically apps using HTTP requests.
+                    The Apify API is built around REST.
+                    The API has predictable resource-oriented URLs, returns JSON-encoded responses,
+                    and uses standard HTTP response codes, authentication, and verbs.
+                        <div>
+                            <Button to='/api/v2'>Check API reference</Button>
+                        </div>
+                    </div>}
+                >
+                    <ClientCodeWrapper>
+                        <CodeBlock content={[{ key: 'cURL', label: 'cURL', language: 'bash', code: `
+                # Prepare Actor input
+
+cat > input.json <<'EOF'
+{
+ // Define the input in JSON here
+}
+EOF
+
+# Run the Actor
+curl "https://api.apify.com/v2/acts/username~actorname/runs?token=<YOUR_API_TOKEN>" \\
+  -X POST \\
+  -d @input.json \\
+  -H 'Content-Type: application/json'
+
+# Use the defaultDatasetId from response and pass it instead of <DATASET_ID>
+
+curl "https://api.apify.com/v2/datasets/<DATASET_ID>/items?token=<YOUR_API_TOKEN>"` }]} hideBashPromptPrefixes hideLineNumbers defaultTabKey="cURL" />
+                    </ClientCodeWrapper>
+                </SectionWrapper>
+                <Section
+                    className={styles.LargerContent}
+                    heading="API client"
+                    description="The official library to interact with Apify API."
+                >
+                    <Tabs items={[
+                        {
+                            title: <TabTitleWrapper>
+                                <ThemedImage
+                                    height={16}
+                                    width={16}
+                                    sources={{ dark: useBaseUrl('/img/javascript-40x40.svg'), light: useBaseUrl('/img/javascript-40x40.svg') }}
+                                />
+                                JavaScript Client
+                            </TabTitleWrapper>,
+                            content: (
+                                <SectionWrapper
+                                    heading="JavaScript API client"
+                                    description={<div className="Description">
+                                    The official library to interact with Apify API from a web browser, Node.js, JavaScript, or Typescript applications.
+                                        <GitHubButton href="https://github.com/apify/apify-client-js" data-color-scheme="no-preference: light; light: light; dark: dark;" data-size="large" data-show-count="true">Star</GitHubButton>
```

Comment:
Seems like the GH button is not picking up the right background in light mode, works on the homepage


<img width="831" alt="Snímek obrazovky 2024-05-05 v 12 28 37" src="https://github.com/apify/apify-docs/assets/45016873/e696599e-e89d-48a8-8ed0-e491613b2834">